src/giftCards/GiftCardCreateDialog/ContentWithProgress.tsx

```diff
@@ -0,0 +1,40 @@
+import { CircularProgress } from "@material-ui/core";
+import { makeStyles } from "@saleor/macaw-ui";
+import classNames from "classnames";
+import React from "react";
+
+interface ContentWithProgressProps {
+  containerClassName?: string;
+  children: React.ReactNode | React.ReactNode[];
+}
+
+export const useStyles = makeStyles(
+  theme => ({
+    container: {
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+      height: "100%",
+      width: "100%",
+      padding: theme.spacing(3)
+    }
+  }),
+  { name: "ContentWithProgress" }
+);
+
+const ContentWithProgress: React.FC<ContentWithProgressProps> = ({
+  containerClassName,
+  children
+}) => {
+  const classes = useStyles({});
+
+  return children ? (
+    (children as React.ReactElement)
```

Comment:
casting not needed

Reply:
Actually it's needed, otherwise typescript fails